### PR TITLE
simplify generic component usage example

### PR DESCRIPTION
**Pull request by chodorowicz**

### CODE REVIEW

**Comment by chodorowicz** [comment]
Of course, sorry for not noticing this contribution guide in the first place 🙇
Fixed, should be good now!

File: playground/src/components/generic-list.usage.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 import { IUser } from '@src/models';
 import { GenericList } from '@src/components';
 
-export const UserList = class extends GenericList<IUser> { };
+export class UserList extends GenericList<IUser> { };
```
Comment (piotrwitek) [inline playground/src/components/generic-list.usage.tsx]: linter screaming on the semicolon after class, shouldn't be there
`yarn lint` in playground folder

**Comment by chodorowicz** [comment]
Ha, OS is not easy :) fixed!
Maybe it'd be good idea to integrate this repo with Travis and have this check automatically? Or add it to Contribution Guide at least.

**Comment by piotrwitek** [comment]
Yes, raised a good point, I will add it to the Contribution Guide and integrate with CI :)
Thanks for suggestion 👍 

PR look's sweet now, thanks for all the hard work! ;)

**Comment by chodorowicz** [comment]
Cool! Thank you for this awesome guide! 🙇

**Pull request merged**